src/App.jsx

```diff
@@ -3,28 +3,28 @@ import { ConnectedRouter } from 'connected-react-router';
 import PropTypes from 'prop-types';
 import SiteHeader from '@edx/frontend-component-header';
 import SiteFooter from '@edx/frontend-component-footer';
-
+import { IntlProvider, injectIntl } from '@edx/frontend-platform/i18n';
 import ConnectedConsolePage from './console/ConsolePage';
 
-function PageContent() {
-  return (
-    <div id="app" className="d-flex flex-column min-vh-100">
-      <SiteHeader />
-      <main className="flex-grow-1">
-        <ConnectedConsolePage />
-      </main>
-      <SiteFooter />
-    </div>
-  );
-}
+const PageContent = () => {
+  <div id="app" className="d-flex flex-column min-vh-100">
+    <SiteHeader />
+    <main className="flex-grow-1">
+      <ConnectedConsolePage />
+    </main>
+    <SiteFooter />
+  </div>;
+};
+
+const IntlPageContent = injectIntl(PageContent);
 
-function App({ history }) {
-  return (
+const App = ({ history }) => {
+  <IntlProvider>
```

Comment:
@Mashal-m Is there a reason IntlProvider was added pack here? It looks like it was intentionally removed in https://github.com/openedx/frontend-app-program-console/pull/114. Just poking around as I'm trying to resolve the React issue

Reply:
Hey @alangsto, I did not add that happened because of conflicts, I guess. I have created a new PR for this work you can look at [this one](https://github.com/openedx/frontend-app-program-console/pull/122)